File: backend/services/comfort_classifier.py
def classify(wrms):
    """Phan loai ISO-2631 comfort level"""
    if wrms < 0.315:
        return "Comfortable"
    elif wrms < 0.63:
        return "Some discomfort"
    elif wrms < 1.0:
        return "Quite uncomfortable"
    elif wrms < 1.6:
        return "Uncomfortable"
    elif wrms < 2.5:
        return "Very uncomfortable"
    else:
        return "Extremely uncomfortable"


def get_color(wrms):
    """6 muc mau ISO-2631, dong bo voi FE comfortGradient."""
    if wrms < 0.315:
        return "#10B981"   # Comfortable — green
    elif wrms < 0.63:
        return "#65A30D"   # Some discomfort — lime
    elif wrms < 1.0:
        return "#F59E0B"   # Quite uncomfortable — amber
    elif wrms < 1.6:
        return "#F97316"   # Uncomfortable — orange
    elif wrms < 2.5:
        return "#EF4444"   # Very uncomfortable — red
    else:
        return "#991B1B"   # Extremely uncomfortable — deep red

File: src/utils/comfortClassifier.js
// Phan loai ISO-2631 theo bang trong readme.md
export function classifyComfort(wrms) {
  if (wrms < 0.315) return 'Comfortable';
  if (wrms < 0.63) return 'Some discomfort';
  if (wrms < 1.0) return 'Quite uncomfortable';
  if (wrms < 1.6) return 'Uncomfortable';
  if (wrms < 2.5) return 'Very uncomfortable';
  return 'Extremely uncomfortable';
}

// Palette dong bo voi comfortGradient (colors.js) — 6 muc chuan ISO-2631.
export function getComfortColor(wrms) {
  if (wrms < 0.315) return '#10B981';   // Comfortable — green
  if (wrms < 0.63)  return '#65A30D';   // Some discomfort — lime
  if (wrms < 1.0)   return '#F59E0B';   // Quite uncomfortable — amber
  if (wrms < 1.6)   return '#F97316';   // Uncomfortable — orange
  if (wrms < 2.5)   return '#EF4444';   // Very uncomfortable — red
  return '#991B1B';                      // Extremely uncomfortable — deep red
}
